Panic

After #521 PR merged, Edge Orchestration container running with DataStorage goes to panic.

### How to Reproduce
Run according to the [instruction](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/docs/datastorage.md#4-how-to-run).

### LOG
```
NFO[2022-03-17T23:36:34Z]discovery.go:574 func1 [deviceDetectionRoutine] edge-orchestration-edd52da3-fcce-4180-aed4-6ff75e18384e
INFO[2022-03-17T23:36:34Z]discovery.go:575 func1 [deviceDetectionRoutine] confInfo    : ExecType(container), Platform(docker)
INFO[2022-03-17T23:36:34Z]discovery.go:576 func1 [deviceDetectionRoutine] netInfo     : IPv4([10.113.171.81]), RTT(0)
INFO[2022-03-17T23:36:34Z]discovery.go:577 func1 [deviceDetectionRoutine] serviceInfo : Services([DataStorage])
INFO[2022-03-17T23:36:34Z]discovery.go:578 func1
panic: interface conversion: interface {} is nil, not string

goroutine 68 [running]:
github.com/lf-edge/edge-home-orchestration-go/internal/controller/storagemgr/config.GetMetadataServerIP({0xcc14d4, 0x7})
        /home/t25kim/work/edge-home-orchestration-go/internal/controller/storagemgr/config/toml.go:189 +0xd6
github.com/lf-edge/edge-home-orchestration-go/internal/controller/storagemgr.checkMetadataStatus()
        /home/t25kim/work/edge-home-orchestration-go/internal/controller/storagemgr/storage.go:89 +0x29
github.com/lf-edge/edge-home-orchestration-go/internal/controller/storagemgr.(*StorageImpl).StartStorage(0xc000304480, {0xc00060c2e0, 0xcb8345})
        /home/t25kim/work/edge-home-orchestration-go/internal/controller/storagemgr/storage.go:123 +0x1c5
github.com/lf-edge/edge-home-orchestration-go/internal/controller/discoverymgr.deviceDetectionRoutine.func1()
        /home/t25kim/work/edge-home-orchestration-go/internal/controller/discoverymgr/discovery.go:595 +0x93a
created by github.com/lf-edge/edge-home-orchestration-go/internal/controller/discoverymgr.deviceDetectionRoutine
        /home/t25kim/work/edge-home-orchestration-go/internal/controller/discoverymgr/discovery.go:550 +0x25
```

--- a/internal/controller/storagemgr/config/toml.go
+++ b/internal/controller/storagemgr/config/toml.go
@@ -88,12 +88,12 @@ type Clients map[string]Client
 
 // Toml contains the struct for building the DataStorage configuration file.
 type Toml struct {
-	Writable
-	Service
-	Registry
-	Device
-	DeviceList
-	Clients
+	Writable   Writable
+	Service    Service
+	Registry   Registry
+	Device     Device
+	DeviceList DeviceList
+	Clients    Clients
 }
 
 var (

